akka-javasdk-tests/src/test/resources/application.conf

```diff
@@ -1,2 +1,3 @@
 # Using a different port to not conflict with parallel tests
 akka.javasdk.testkit.http-port = 39391
+akka.javasdk.components.disabled = "akkajavasdk.components.keyvalueentities.user.ProdCounterEntity,akkajavasdk.components.keyvalueentities.user.StageCounterEntity"
```

Comment:
is this used by any test? it is wrong.
`disabled` vs `disable`

Reply:
just noticed :) fixed